_inc/footer.php

```diff
@@ -5,20 +5,20 @@
 				<div class="a8c-attribution">
 					<span>
 						<?php echo sprintf( __( 'An %s Airline', 'jetpack' ),
-						'<a href="http://automattic.com/" class="a8c-logo" target="_blank">Automattic</a>'
+						'<a href="https://automattic.com/" class="a8c-logo" target="_blank">Automattic</a>'
 						); ?>
 					</span>
 				</div>
 			</nav><!-- .primary -->
 
 			<nav class="secondary nav-horizontal">
 				<div class="secondary-footer">
-					<a href="http://jetpack.com" target="_blank">Jetpack <?php echo JETPACK__VERSION; ?></a>
-					<a href="http://wordpress.com/tos/" target="_blank"><?php esc_html_e( 'Terms', 'jetpack' ); ?></a>
-					<a href="http://automattic.com/privacy/" target="_blank"><?php esc_html_e( 'Privacy', 'jetpack' ); ?></a>
+					<a href="https://jetpack.com" target="_blank">Jetpack <?php echo JETPACK__VERSION; ?></a>
+					<a href="https://en.wordpress.com/tos/" target="_blank"><?php esc_html_e( 'Terms', 'jetpack' ); ?></a>
```

Comment:
For the TOS, we should leave it without the language subdomain. There is code in place to redirect to other languages based on various bits, e.g https://de.wordpress.com/tos/

They will all be https though.